aten/src/ATen/cpu/vec/vec512/vec512_int.h

```diff
@@ -918,6 +1208,12 @@ Vectorized<int8_t> inline operator*(const Vectorized<int8_t>& a, const Vectorize
   return int_elementwise_binary_512(a, b, std::multiplies<int8_t>());
 }
 
+template <>
+Vectorized<uint8_t> inline operator*(const Vectorized<uint8_t>& a, const Vectorized<uint8_t>& b) {
+  // We don't have an instruction for multiplying uint8_t
+  return int_elementwise_binary_512(a, b, std::multiplies<uint8_t>());
+}
+
```

Comment:
Would https://stackoverflow.com/q/67243567 be appropriate here? Thanks!

cc @jgong5 for his opinion

Reply:
Please note that vectorized uint8_t code is modeled after int8_t.  What you suggested would be better indeed.  I did alike for shift operators, and can do it for multiplication too, but this is out of scope of this stack of PRs.  So I'll try to make the change at some later point, if no one else does it in the meantime.

Reply:
Sure, thanks, @alexsamardzic!